tests/integration/CommandTesterTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Robo;
+
+use PHPUnit\Framework\TestCase;
+use Robo\Traits\CommandTesterTrait;
+use RoboExample\Robo\Plugin\Commands\ExampleCommands;
+
+class CommandTestertTest extends TestCase
+{
+    use CommandTesterTrait;
+
+    public function setUp()
+    {
+        $this->setupCommandTester(ExampleCommands::class);
+    }
+
+    public function testInputApis()
+    {
+        list($tryInputOutput, $statusCode) = $this->executeCommand('try:input', ["I'm great!", "yes", "PHP", "1234"]);
+        $this->assertEquals(0, $statusCode);
+        $this->assertContains("I'm great!", $tryInputOutput);
+        $this->assertContains("PHP", $tryInputOutput);
+        $this->assertContains("1234", $tryInputOutput);
+    }
+
+    public function testTesterWithOptions()
+    {
+        list($execOutput, $statusCode) = $this->executeCommand('try:exec', []);
+        $this->assertEquals(0, $statusCode);
+        list($execOutput, $statusCode) = $this->executeCommand('try:exec', [], ['--error']);
+        $this->assertNotEquals(0, $statusCode);
+    }
```

Comment:
One thing of note, and I don't quite understand it, but this test generates the following output:

```
lpeabody@pop-os:~/Projects/Robo (feature/947-part-deux-build-test-application)$ fin exec /var/www/vendor/bin/phpunit tests/integration/CommandTesterTest.php
PHPUnit 6.5.14 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.3.19
Configuration: /var/www/phpunit.xml.dist
Error:         No code coverage driver is available

.hey
hey
hey
hey
hoy
hoy
hoy
gou
gou
die
.                                                                                                                                                                    2 / 2 (100%)

Time: 10.23 seconds, Memory: 8.00MB

OK (2 tests, 6 assertions)
```

I assume this is because there is some disconnect between the output stream injected into the command class versus the output stream available to each ExecTask being executed?

Reply:
That output, `hey`, `hoy`, etc. is what one of the test fixture scripts print out. It seems that the fixture is printing its output to stdout instead of having it captured.

I don't see that when I run `composer unit tests/integration/CommandTesterTest.php`; maybe it has something to do with how you're running the tests? Does `composer unit` work for you?

Reply:
I ran the test by explicitly calling `/var/www/vendor/bin/phpunit tests/integration/CommandTesterTest.php`. However, if I run `composer run unit` I do not see the above mentioned output.